qiime2/core/path.py

```diff
@@ -92,14 +92,20 @@ def _destruct(cls, path):
         else:
             os.unlink(path)
 
-    def __new__(cls, dir=False, **kwargs):
```

Comment:
Where there any callers of this? I swear we set `prefix` in a lot of places for stuff like `q2-provenance`... although come to think of it, cache has probably eroded out most of those callsites, but there may still be a few left.